### DISCUSSION TITLE
[HELP] charon.log Restart router A. The process is successfully started and the configuration parameters are correct. However, the connection to IPsec server B fails.

### DISCUSSION THREAD

**Discussion by aigenisi-aliyi** [strongswan/strongswan#2068] [General]
00[DMN] Starting IKE charon daemon (strongSwan 5.9.10, Linux 5.10.176+, aarch64)
00[LIB] plugin 'random': loaded successfully
00[LIB] plugin 'nonce': loaded successfully
00[LIB] plugin 'pubkey': failed to load - pubkey_plugin_create not found and no plugin file available
00[LIB] plugin 'counters': failed to load - counters_plugin_create not found and no plugin file available
00[LIB] plugin 'hmac': loaded successfully
00[LIB] plugin 'pem': loaded successfully
00[LIB] plugin 'openssl': loaded successfully
00[LIB] plugin 'kernel-netlink': loaded successfully
00[LIB] plugin 'socket-default': loaded successfully
00[LIB] plugin 'updown': loaded successfully
00[LIB] plugin 'vici': loaded successfully
00[KNL] received netlink error: Unknown device type (95)
00[KNL] failed to create XFRM interface 'xfrmi-test-1518'
00[KNL] known interfaces and IP addresses:
00[KNL]   lo
00[KNL]     127.0.0.1
00[KNL]     ::1
00[KNL]   eth_x
00[KNL]     5.5.7.157
00[KNL]     fe80::45e:23aa:d14e:9ac
00[KNL]   eth0
00[KNL]     fe80::ca14:e34d:b010:5d46
00[KNL]   rmnet2
00[KNL]     fe80::ba2c:95a1:ed86:c510
00[KNL]   br0
00[KNL]     192.168.8.1
00[KNL]     fd62:986a:d950:6600:6934:9329:21a7:7dc
00[KNL]     fe80::92fa:96f1:b485:ba38
00[LIB] feature PRIVKEY:DSA in plugin 'pem' has unmet dependency: PRIVKEY:DSA
00[LIB] feature PRIVKEY:BLISS in plugin 'pem' has unmet dependency: PRIVKEY:BLISS
00[LIB] feature PUBKEY:DSA in plugin 'pem' has unmet dependency: PUBKEY:DSA
00[LIB] feature PUBKEY:BLISS in plugin 'pem' has unmet dependency: PUBKEY:BLISS
00[LIB] feature CERT_DECODE:PGP in plugin 'pem' has unmet dependency: CERT_DECODE:PGP
00[LIB] feature CERT_DECODE:OCSP_REQUEST in plugin 'pem' has unmet dependency: CERT_DECODE:OCSP_REQUEST
00[LIB] feature CERT_DECODE:OCSP_RESPONSE in plugin 'pem' has unmet dependency: CERT_DECODE:OCSP_RESPONSE
00[LIB] feature CERT_DECODE:X509_AC in plugin 'pem' has unmet dependency: CERT_DECODE:X509_AC
00[LIB] feature CERT_DECODE:PKCS10_REQUEST in plugin 'pem' has unmet dependency: CERT_DECODE:PKCS10_REQUEST
00[LIB] feature CERT_DECODE:PUBKEY in plugin 'pem' has unmet dependency: CERT_DECODE:PUBKEY
00[LIB] feature CONTAINER_DECODE:PKCS7 in plugin 'pem' has unmet dependency: CONTAINER_DECODE:PKCS7
00[LIB] loaded plugins: charon random nonce hmac pem openssl kernel-netlink socket-default updown vici
00[LIB] unable to load 11 plugin features (11 due to unmet dependencies)
00[LIB] dropped capabilities, running as uid 2423, gid 2423
00[JOB] spawning 16 worker threads
01[LIB] created thread 01 [2001]
02[LIB] created thread 02 [2002]
03[LIB] created thread 03 [2003]
04[LIB] created thread 04 [2000]
05[LIB] created thread 05 [2004]
06[LIB] created thread 06 [1999]
07[LIB] created thread 07 [1998]
08[LIB] created thread 08 [2005]
09[LIB] created thread 09 [1997]
10[LIB] created thread 10 [1996]
11[LIB] created thread 11 [1995]
12[LIB] created thread 12 [1994]
13[LIB] created thread 13 [1993]
14[LIB] created thread 14 [1992]
15[LIB] created thread 15 [1991]
16[LIB] created thread 16 [1990]
13[CFG] vici client 1 connected
06[CFG] vici client 1 requests: get-keys
13[CFG] vici client 1 requests: get-shared
09[CFG] vici client 1 requests: load-shared
09[CFG] loaded IKE shared key with id 'ike-n2n-1' for: '5.5.7.157', '5.5.6.63'
16[CFG] vici client 1 requests: get-authorities
11[CFG] vici client 1 requests: get-pools
05[CFG] vici client 1 requests: get-conns
14[CFG] vici client 1 requests: load-conn
14[CFG]  conn n2n-1:
14[CFG]   child n2n-1:
14[CFG]    rekey_time = 3600
14[CFG]    life_time = 3960
14[CFG]    rand_time = 360
14[CFG]    rekey_bytes = 0
14[CFG]    life_bytes = 0
14[CFG]    rand_bytes = 0
14[CFG]    rekey_packets = 0
14[CFG]    life_packets = 0
14[CFG]    rand_packets = 0
14[CFG]    updown = /system/bin/ipsec/_updown iptables
14[CFG]    hostaccess = 0
14[CFG]    ipcomp = 0
14[CFG]    mode = TUNNEL
14[CFG]    policies = 1
14[CFG]    policies_fwd_out = 0
14[CFG]    dpd_action = none
14[CFG]    start_action = start
14[CFG]    close_action = none
14[CFG]    reqid = 0
14[CFG]    tfc = 0
14[CFG]    priority = 0
14[CFG]    interface = (null)
14[CFG]    if_id_in = 0
14[CFG]    if_id_out = 0
14[CFG]    mark_in = 0/0
14[CFG]    mark_in_sa = 0
14[CFG]    mark_out = 0/0
14[CFG]    set_mark_in = 0/0
14[CFG]    set_mark_out = 0/0
14[CFG]    label = (null)
14[CFG]    label_mode = system
14[CFG]    inactivity = 0
14[CFG]    proposals = ESP:AES_CBC_256/HMAC_SHA1_96/MODP_2048/NO_EXT_SEQ, ESP:AES_CBC_192/HMAC_SHA1_96/MODP_2048/NO_EXT_SEQ, ESP:AES_CBC_128/HMAC_SHA1_96/MODP_2048/NO_EXT_SEQ
14[CFG]    local_ts = 192.168.8.0/24
14[CFG]    remote_ts = 192.168.1.0/24
14[CFG]    hw_offload = no
14[CFG]    sha256_96 = 0
14[CFG]    copy_df = 1
14[CFG]    copy_ecn = 1
14[CFG]    copy_dscp = out
14[CFG]   version = 2
14[CFG]   local_addrs = 5.5.7.157
14[CFG]   remote_addrs = 5.5.6.63
14[CFG]   local_port = 500
14[CFG]   remote_port = 500
14[CFG]   send_certreq = 1
14[CFG]   send_cert = CERT_SEND_IF_ASKED
14[CFG]   ppk_id = (null)
14[CFG]   ppk_required = 0
14[CFG]   mobike = 1
14[CFG]   aggressive = 0
14[CFG]   dscp = 0x00
14[CFG]   encap = 0
14[CFG]   dpd_delay = 10
14[CFG]   dpd_timeout = 0
14[CFG]   fragmentation = 0
14[CFG]   childless = 0
14[CFG]   unique = UNIQUE_NO
14[CFG]   keyingtries = 1
14[CFG]   reauth_time = 0
14[CFG]   rekey_time = 3600
14[CFG]   over_time = 360
14[CFG]   rand_time = 360
14[CFG]   proposals = IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_192/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_128/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048
14[CFG]   if_id_in = 0
14[CFG]   if_id_out = 0
14[CFG]   local:
14[CFG]    id = 5.5.7.157
14[CFG]    class = pre-shared key
14[CFG]   remote:
14[CFG]    id = 5.5.6.63
14[CFG]    class = pre-shared key
14[CFG] added vici connection: n2n-1
14[CFG] initiating 'n2n-1'
14[MGR] checkout IKE_SA by config
14[MGR] created IKE_SA (unnamed)[1]
14[IKE] <n2n-1|1> queueing IKE_VENDOR task
14[IKE] <n2n-1|1> queueing IKE_INIT task
14[IKE] <n2n-1|1> queueing IKE_NATD task
14[IKE] <n2n-1|1> queueing IKE_CERT_PRE task
14[IKE] <n2n-1|1> queueing IKE_AUTH task
14[IKE] <n2n-1|1> queueing IKE_CERT_POST task
14[IKE] <n2n-1|1> queueing IKE_CONFIG task
14[IKE] <n2n-1|1> queueing IKE_AUTH_LIFETIME task
14[IKE] <n2n-1|1> queueing IKE_MOBIKE task
14[IKE] <n2n-1|1> queueing IKE_ESTABLISH task
14[IKE] <n2n-1|1> queueing CHILD_CREATE task
14[IKE] <n2n-1|1> activating new tasks
14[IKE] <n2n-1|1>   activating IKE_VENDOR task
14[IKE] <n2n-1|1>   activating IKE_INIT task
14[IKE] <n2n-1|1>   activating IKE_NATD task
14[IKE] <n2n-1|1>   activating IKE_CERT_PRE task
14[IKE] <n2n-1|1>   activating IKE_AUTH task
14[IKE] <n2n-1|1>   activating IKE_CERT_POST task
14[IKE] <n2n-1|1>   activating IKE_CONFIG task
14[IKE] <n2n-1|1>   activating IKE_AUTH_LIFETIME task
14[IKE] <n2n-1|1>   activating IKE_MOBIKE task
14[IKE] <n2n-1|1>   activating IKE_ESTABLISH task
14[IKE] <n2n-1|1>   activating CHILD_CREATE task
14[IKE] <n2n-1|1> initiating IKE_SA n2n-1[1] to 5.5.6.63
14[IKE] <n2n-1|1> IKE_SA n2n-1[1] state change: CREATED => CONNECTING
14[CFG] <n2n-1|1> configured proposals: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_192/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_128/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048
14[CFG] <n2n-1|1> sending supported signature hash algorithms: sha256 sha384 sha512 identity
14[ENC] <n2n-1|1> generating IKE_SA_INIT request 0 [ SA KE No N(NATD_S_IP) N(NATD_D_IP) N(HASH_ALG) N(REDIR_SUP) ]
14[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (544 bytes)
14[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 0000000000000000_r
14[MGR] <n2n-1|1> checkin of IKE_SA successful
05[CFG] vici client 1 disconnected
15[MGR] checkout IKEv2 SA by message with SPIs 775325d35c575752_i 51e453bb47de099f_r
15[MGR] IKE_SA n2n-1[1] successfully checked out
15[NET] <n2n-1|1> received packet: from 5.5.6.63[500] to 5.5.7.157[500] (381 bytes)
15[ENC] <n2n-1|1> parsed IKE_SA_INIT response 0 [ SA KE No V ]
15[IKE] <n2n-1|1> received Cisco FlexVPN Supported vendor ID
15[CFG] <n2n-1|1> selecting proposal:
15[CFG] <n2n-1|1>   proposal matches
15[CFG] <n2n-1|1> received proposals: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048
15[CFG] <n2n-1|1> configured proposals: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_192/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048, IKE:AES_CBC_128/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048
15[CFG] <n2n-1|1> selected proposal: IKE:AES_CBC_256/HMAC_SHA1_96/PRF_HMAC_SHA1/MODP_2048
15[IKE] <n2n-1|1> reinitiating already active tasks
15[IKE] <n2n-1|1>   IKE_CERT_PRE task
15[IKE] <n2n-1|1>   IKE_AUTH task
15[IKE] <n2n-1|1> authentication of '5.5.7.157' (myself) with pre-shared key
15[IKE] <n2n-1|1> successfully created shared key MAC
15[CFG] <n2n-1|1> proposing traffic selectors for us:
15[CFG] <n2n-1|1>  192.168.8.0/24
15[CFG] <n2n-1|1> proposing traffic selectors for other:
15[CFG] <n2n-1|1>  192.168.1.0/24
15[CFG] <n2n-1|1> configured proposals: ESP:AES_CBC_256/HMAC_SHA1_96/NO_EXT_SEQ, ESP:AES_CBC_192/HMAC_SHA1_96/NO_EXT_SEQ, ESP:AES_CBC_128/HMAC_SHA1_96/NO_EXT_SEQ
15[IKE] <n2n-1|1> establishing CHILD_SA n2n-1{1}
15[KNL] <n2n-1|1> got SPI c5ccf036
15[ENC] <n2n-1|1> generating IKE_AUTH request 1 [ IDi N(INIT_CONTACT) IDr AUTH SA TSi TSr N(MOBIKE_SUP) N(ADD_4_ADDR) N(ADD_6_ADDR) N(EAP_ONLY) N(MSG_ID_SYN_SUP) ]
15[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (364 bytes)
15[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
15[MGR] <n2n-1|1> checkin of IKE_SA successful
16[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 0000000000000000_r
16[MGR] IKE_SA n2n-1[1] successfully checked out
16[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
16[MGR] <n2n-1|1> checkin of IKE_SA successful
07[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
07[MGR] IKE_SA n2n-1[1] successfully checked out
07[IKE] <n2n-1|1> retransmit 1 of request with message ID 1
07[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (364 bytes)
07[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
07[MGR] <n2n-1|1> checkin of IKE_SA successful
14[KNL] interface wlan5 activated
09[KNL] creating roam job due to address/link change
09[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
09[MGR] IKE_SA n2n-1[1] successfully checked out
09[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
09[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
09[MGR] <n2n-1|1> checkin of IKE_SA successful
06[KNL] interface wlan0 activated
10[KNL] interface wlan3 activated
06[KNL] creating roam job due to address/link change
06[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] IKE_SA n2n-1[1] successfully checked out
06[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
06[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] <n2n-1|1> checkin of IKE_SA successful
14[KNL] interface wlan3 deactivated
12[KNL] creating roam job due to address/link change
12[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
12[MGR] IKE_SA n2n-1[1] successfully checked out
12[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
12[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
12[MGR] <n2n-1|1> checkin of IKE_SA successful
06[KNL] fe80::6286:6222:ae76:6dc3 appeared on wlan0
14[KNL] creating roam job due to address/link change
14[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
14[MGR] IKE_SA n2n-1[1] successfully checked out
14[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
14[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
14[MGR] <n2n-1|1> checkin of IKE_SA successful
10[KNL] fe80::6126:ace0:1964:3728 appeared on wlan5
16[KNL] creating roam job due to address/link change
16[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
16[MGR] IKE_SA n2n-1[1] successfully checked out
16[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
16[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
16[MGR] <n2n-1|1> checkin of IKE_SA successful
07[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
07[MGR] IKE_SA n2n-1[1] successfully checked out
07[IKE] <n2n-1|1> retransmit 2 of request with message ID 1
07[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (364 bytes)
07[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
07[MGR] <n2n-1|1> checkin of IKE_SA successful
14[KNL] interface wlan2 activated
13[KNL] interface wlan4 activated
05[KNL] creating roam job due to address/link change
05[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
05[MGR] IKE_SA n2n-1[1] successfully checked out
05[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
05[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
05[MGR] <n2n-1|1> checkin of IKE_SA successful
10[KNL] interface wlan1 activated
06[KNL] interface wlan7 activated
10[KNL] creating roam job due to address/link change
10[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] IKE_SA n2n-1[1] successfully checked out
10[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
10[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] <n2n-1|1> checkin of IKE_SA successful
13[KNL] interface wlan6 activated
14[KNL] interface wlan8 activated
16[KNL] creating roam job due to address/link change
16[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
16[MGR] IKE_SA n2n-1[1] successfully checked out
16[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
16[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
16[MGR] <n2n-1|1> checkin of IKE_SA successful
10[KNL] fe80::fa7b:834f:3e80:6e3f appeared on wlan4
13[KNL] creating roam job due to address/link change
13[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
13[MGR] IKE_SA n2n-1[1] successfully checked out
13[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
13[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
13[MGR] <n2n-1|1> checkin of IKE_SA successful
06[KNL] fe80::694f:75ea:492f:ef22 appeared on wlan2
14[KNL] fe80::922b:a783:71b7:9f69 appeared on wlan8
10[KNL] creating roam job due to address/link change
10[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] IKE_SA n2n-1[1] successfully checked out
10[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
10[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] <n2n-1|1> checkin of IKE_SA successful
13[KNL] fe80::1727:912b:c9e9:850e appeared on wlan6
06[KNL] creating roam job due to address/link change
06[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] IKE_SA n2n-1[1] successfully checked out
06[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
06[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] <n2n-1|1> checkin of IKE_SA successful
14[KNL] fe80::8a11:d5fa:a007:5ea5 appeared on wlan1
10[KNL] creating roam job due to address/link change
10[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] IKE_SA n2n-1[1] successfully checked out
10[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
10[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
10[MGR] <n2n-1|1> checkin of IKE_SA successful
13[KNL] fe80::f122:a7a1:c357:fd50 appeared on wlan7
06[KNL] creating roam job due to address/link change
06[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] IKE_SA n2n-1[1] successfully checked out
06[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
06[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
06[MGR] <n2n-1|1> checkin of IKE_SA successful
09[KNL] interface wlan2 deactivated
16[KNL] fe80::694f:75ea:492f:ef22 disappeared from wlan2
06[KNL] interface wlan4 deactivated
10[KNL] fe80::fa7b:834f:3e80:6e3f disappeared from wlan4
11[KNL] interface wlan1 deactivated
05[KNL] fe80::8a11:d5fa:a007:5ea5 disappeared from wlan1
07[KNL] interface wlan7 deactivated
12[KNL] fe80::f122:a7a1:c357:fd50 disappeared from wlan7
13[KNL] interface wlan6 deactivated
14[KNL] fe80::1727:912b:c9e9:850e disappeared from wlan6
08[KNL] creating roam job due to address/link change
08[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] IKE_SA n2n-1[1] successfully checked out
08[IKE] <n2n-1|1> keeping statically configured path 5.5.7.157 - 5.5.6.63
08[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] <n2n-1|1> checkin of IKE_SA successful
08[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] IKE_SA n2n-1[1] successfully checked out
08[IKE] <n2n-1|1> retransmit 3 of request with message ID 1
08[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (364 bytes)
08[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] <n2n-1|1> checkin of IKE_SA successful
08[MGR] checkout IKEv2 SA with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] IKE_SA n2n-1[1] successfully checked out
08[IKE] <n2n-1|1> retransmit 4 of request with message ID 1
08[NET] <n2n-1|1> sending packet: from 5.5.7.157[500] to 5.5.6.63[500] (364 bytes)
08[MGR] <n2n-1|1> checkin IKEv2 SA n2n-1[1] with SPIs 775325d35c575752_i 51e453bb47de099f_r
08[MGR] <n2n-1|1> checkin of IKE_SA successful

**Comment by aigenisi-aliyi** [comment]
this title "[HELP] charon.log Restart" restart is reboot

**Comment by tobiasbrunner** [comment]
The first message apparently doesn't get through. Which could be due to a lot of reasons, wrong IP configured, firewall blocking the traffic (UDP ports 500 and 4500, plus ESP if there is no NAT), general connectivity issue, no IKE server running at the target IP, wrong IKE version or some other configuration issue at the other end that causes it to not respond etc.
